docs/FAQ.md

```diff
@@ -98,3 +98,8 @@ Some use cases:
 - Show/hide some inputs if the current form values matches specific constraints
 
 For all those cases, you can use the [aor-dependent-input](https://github.com/marmelab/aor-dependent-input) addon.
+
+## UI in production build is empty or broke
```

Comment:
I must admit I don't know how to describe best the error, if you have ideas tell me.

Reply:
I've never experienced it myself. I don't know what an "empty build" is. A white page? And does a "broken build" show an error in the console, and if so, which one?

Reply:
CSS classes are mixed. The visual result is really random, from a white page to a broken UI.
And it can change every time you refresh the page. Hard to tell.